lib/matplotlib/patches.py

```diff
@@ -3094,10 +3094,10 @@ def ensure_quadratic_bezier(path):
             its control points if true.
             """
             segments = list(path.iter_segments())
-            assert len(segments) == 2
-
-            assert segments[0][1] == Path.MOVETO
-            assert segments[1][1] == Path.CURVE3
+            if (len(segments) != 2) or (segments[0][1] != Path.MOVETO) or\
+               (segments[1][1] != Path.CURVE3):
```

Comment:
Aligning with the inner block is a greater style issue that over indenting, can you push this in a few more spaces?